package.json

```diff
@@ -50,7 +51,7 @@
   },
   "scripts": {
     "build": "rollup -c",
-    "autobuild": "rollup -c -w",
+    "autobuild": "rollup -c -w NODE_ENV:production",
```

Comment:
I think this requires the `--environment` flag

Reply:
oops, need to remove it, not actually necessary